doc/tools/docgram/fullGrammar

```diff
@@ -1665,8 +1665,6 @@ simple_tactic: [
 | "generalize_eqs_vars" hyp
 | "dependent" "generalize_eqs_vars" hyp
 | "specialize_eqs" hyp
-| "hresolve_core" "(" ident ":=" constr ")" "at" nat_or_var "in" constr
-| "hresolve_core" "(" ident ":=" constr ")" "in" constr
 | "hget_evar" nat_or_var
```

Comment:
This should be removed, and similarly in orderedGrammar.

Reply:
Run `make doc_gram_rst` to update this.  (You failed lint for the lack of this step).